### DISCUSSION TITLE
Menu Editing Error

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [AzuraCast/AzuraCast#6690] [General]
Before I used it in the menu it would be
station.php

'configure' => [
                     'label' => __('Configure Server'),
                 'icon' => 'edit_calendar',
                 'url' => $props['nowPlayingUri'] = 'https://link?idpanel=' . urlencode($station->getDescription()),
                 'permission' => StationPermissions::Media,
             ],

where I defined a hash in the description in Azuracast.

However, in the new version the menu was changed to menu.ts
and changed the way it is done

{
             key: 'configure',
             label: computed(() => $gettext('configurar')),
             icon: 'edit_calendar',
             url: {
                 name: 'https://link?idpanel=stations'
             },
             visible: userCanManageMedia && stationProps.features.media,
         },

I tried several ways but the menu doesn't even appear after compiling
Can anyone help me with how to do this?
I need that after idpanel= it shows the station id or the station description, which would be ideal.

**Comment by BusterNeece** [comment]
@Marcioghiraldelli Sorry, I've been meaning to reply to you but I haven't had the chance. In latest versions, as we built out the new portions of the app, we removed the ability to control the menu via PHP code. If you're still using that feature, we can add it back if necessary, but that'll take some work on our part, and I'll have to let you know when it's done.

**Reply by Marcioghiraldelli** [reply]
@BusterNeece 
Thank you very much for responding.
I know your time is short...
I was actually a little discouraged by the change in the menu, because in my current version I use links in the menu, I even edited the menu, removing some functions to make my daily life easier, so this part of being able to edit the menu is great, because You could make the menu any way you liked.
Currently I can't do this as mentioned above.
If I could return to the way it was before, that would be great!!!
I thank you immensely..

**Reply by Marcioghiraldelli** [reply]
hello @BusterNeece 
Do you have more or less an idea of when this change will take place?

**Comment by Marcioghiraldelli** [comment]
Hello, a few weeks ago I asked if it would still be possible to change the menu to insert, for example, external links or even use internal strings as a description of the radio in a menu link.
Before it was station.php and now it isn't.
I can even insert options but not like before.

If there are any costs associated with this change, please let me know.
Please let me know if it is no longer possible.
I thank.
